app/anim.js

```diff
@@ -0,0 +1,60 @@
+
+function myFunction(){
+	document.body.scrollTop = document.documentElement.scrollTop = 0;
+	document.forms['regform'].elements['regname'].focus();
+}
+
+
+
+
+function showPlaylist2() {
+  var appear;
+  appear =document.getElementById("playlist2");
+  appear.style.display = 'block';
+  }
+
+	function hidePlaylist2() {
```

Comment:
Fix identation and stylistic issues: spaces around operator etc. I just cannot read this file